app/SQL/install.sql.mysql.php

```diff
@@ -55,12 +55,14 @@
 	`id_feed` INT,	-- 1.20.0
 	`tags` VARCHAR(2048),
 	`attributes` TEXT,	-- v1.20.0
+	`lastUserModified` BIGINT DEFAULT 0,
 	PRIMARY KEY (`id`),
 	FOREIGN KEY (`id_feed`) REFERENCES `_feed`(`id`) ON DELETE CASCADE ON UPDATE CASCADE,
 	UNIQUE KEY (`id_feed`,`guid`),	-- v0.7
 	INDEX (`is_favorite`),	-- v0.7
 	INDEX (`is_read`),	-- v0.7
 	INDEX `entry_lastSeen_index` (`lastSeen`),	-- v1.1.1
+	INDEX `entry_last_user_modified_index` (`lastUserModified`),	-- v1.1.1
```

Comment:
```suggestion
	INDEX `entry_last_user_modified_index` (`lastUserModified`),	-- v1.27.1
```

and in other places where v1.1.1 is used instead of 1.27.1